src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+
+                            } else if(numTasks.getEventNum()==j+1){
+                                System.out.print(j + 1 + "." + "[" + "E" + "]" + "[" + t.getStatusIcon() + "] " + itemList[j] + "\n");
+
+                            } else {
+                                System.out.print(j + 1 + "." + "[" + " " + "]" + "[" + t.getStatusIcon() + "] " + itemList[j] + "\n");
+
+                            }
+                        }
+                    }
+                    i-=1;
+
+                    System.out.print("____________________________________________________________\n");
+
+                } else if(doneSide[0].equals("done")){
+                    Task t = new Task(itemList[i]);
+                    t.markAsDone();
+                    int taskNum=Integer.parseInt(doneSide[1]);
+
+                    System.out.print("____________________________________________________________\n");
+                    System.out.print("Nice! I've marked this task as done: \n");
+                    System.out.print("\t["+t.getStatusIcon()+"] "+itemList[taskNum-1]+"\n");
+                    System.out.print("____________________________________________________________\n");
+                    numTasks.setNumTasks(taskNum);
+
+                } else if(task[0].equals("todo")){
+                    numTasks.setTaskCount(taskCounts+=1);
+                    String taskItem= line.substring(5);
+                    Todo t = new Todo(taskItem);
+
+                    System.out.print("____________________________________________________________\n");
+                    System.out.print("Got it. I've added this task: \n");
+                    System.out.print("\t"+t.toString());
+                    System.out.print( "Now you have "+numTasks.getTaskCount()+" tasks in the list.");
+                    System.out.print("\n____________________________________________________________\n");
+                    itemList[i] = taskItem;
+                    numTasks.setTodoNum(numTasks.getTaskCount());
+
+                } else if(task[0].equals("deadline")){
+                    numTasks.setTaskCount(taskCounts+=1);
+                    String taskItem= line.substring(9);
+                    Deadline d = new Deadline(taskItem,"");
+
+                    System.out.print("____________________________________________________________\n");
+                    System.out.print("Got it. I've added this task: \n");
+                    System.out.print("\t"+d.toString());
+                    System.out.print( "Now you have "+numTasks.getTaskCount()+" tasks in the list.");
+                    System.out.print("\n____________________________________________________________\n");
+                    itemList[i] = taskItem;
+                    numTasks.setDeadlineNum(numTasks.getTaskCount());
+
+                }
+                else if(task[0].equals("event")){
+                    numTasks.setTaskCount(taskCounts+=1);
```

Comment:
Perhaps this (and the conditions above) would be a good section of code to SLAP?